inference-engine/ie_bridges/python/sample/object_detection_sample_ssd/README.md

```diff
@@ -1,74 +1,119 @@
-# Object Detection Python* Sample SSD {#openvino_inference_engine_ie_bridges_python_sample_object_detection_sample_ssd_README}
+# Python* Object Detection Sample SSD {#openvino_inference_engine_ie_bridges_python_sample_object_detection_sample_ssd_README}
 
-This sample demonstrates how to run the Object Detection sample application.
+This sample demonstrates how to do inference of object detection networks using Synchronous Inference Request API.  
+Models with 1 input and 1 or 2 outputs are supported.  
+In the last case names of output blobs must be "boxes" and "labels".
 
-The sample demonstrates how to use the new Infer Request API of Inference Engine in applications.
-Refer to [Integrate the Inference Engine New Request API with Your Application](../../../../../docs/IE_DG/Integrate_with_customer_application_new_API.md) for details.
-The sample demonstrates how to build and execute an inference request on example of object detection networks.
+The following Inference Engine Python API is used in the application:
 
-Due to properties of SSD networks, this sample works correctly only on a batch of the size 1. For a greater number of images in a batch, network reshape is required.
+| Feature                  | API                                                                                                                         | Description                                           |
+| :----------------------- | :-------------------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------- |
+| Basic Infer Flow         | [IECore], [IECore.read_network], [IECore.load_network]                                                                      | Common API to do inference                            |
+| Synchronous Infer        | [ExecutableNetwork.infer]                                                                                                   | Do synchronous inference                              |
+| Network Operations       | [IENetwork.input_info], [IENetwork.outputs], [InputInfoPtr.precision], [DataPtr.precision], [InputInfoPtr.input_data.shape] | Managing of network: configure input and output blobs |
+| Custom Extension Kernels | [IECore.add_extension], [IECore.set_config]                                                                                 | Load extension library and config to the device       |
```

Comment:
Remove duplication API. Use something like this `Basic Inference Engine API is covered by [Hello Classification Python sample](../hello_classification/README.md)` like in C samples https://github.com/openvinotoolkit/openvino/pull/4780